debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/ParallelSnapshotReader.java

```diff
@@ -33,6 +33,8 @@
     private final AtomicBoolean running = new AtomicBoolean();
     private final AtomicReference<Runnable> uponCompletion = new AtomicReference<>();
 
+    private static final long NEW_TABLES_SERVER_ID_OFFSET = 10000;
```

Comment:
Should we make this configurable? Chances for collision are small, but better to have a way to completely rule it out probably.